Task 1/script.js

```diff
@@ -8,3 +8,11 @@ pamatyti jo pateikto svorio kovertavimą į:
 Pastaba: atvaizdavimas turi būti matomas pateikus formą ir pateikiamas
 <div id="output"></div> viduje, bei turi turėti bent minimalų stilių;
 ------------------------------------------------------------------- */
+
+document.getElementById("search").addEventListener("input", function(e) {
```

Comment:
This should work on submit not on every change. 
Also preventDefault does not do anything here if you would click 'Calculate' it would reload the page.